sonar-php-plugin/src/main/java/org/sonar/plugins/php/phpunit/CompatibilityImportersFactory.java

```diff
@@ -44,19 +42,12 @@ public CompatibilityImportersFactory(SensorContext context) {
   }
 
   public List<ReportImporter> createCoverageImporters() {
-    if (!supportsMultiPathCoverage()) {
-      return createLegacyImporters();
-    }
     if (LEGACY_PATH_KEYS.stream().anyMatch(this::isPropertyUsed) && !multiPathCoverageUsed()) {
       return createLegacyImporters();
     }
     return createMultiCoverageImporter();
   }
 
-  private boolean supportsMultiPathCoverage() {
-    return context.getSonarQubeVersion().isGreaterThanOrEqual(Version.create(6, 2));
-  }
-
   private boolean multiPathCoverageUsed() {
     return context.settings().getStringArray(PhpPlugin.PHPUNIT_COVERAGE_REPORT_PATHS_KEY).length > 0;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "settings"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)